src/status_im/contexts/communities/events.cljs

```diff
@@ -478,3 +508,28 @@
 (rf/reg-event-fx :communities/update-last-opened-at-success
  (fn [{:keys [db]} [community-id last-opened-at]]
    {:db (assoc-in db [:communities community-id :last-opened-at] last-opened-at)}))
+
+(rf/defn share-community
+  {:events [::share-community-confirmation-pressed]}
+  [cofx users community-id]
+  (when (seq users)
+    {:json-rpc/call [{:method      "wakuext_shareCommunity"
+                      :params      [{:communityId community-id
+                                     :users       (vec users)}]
```

Comment:
Are you sure you need to convert to a vector? AFAIR sequences will be correctly encoded to JSON without us having to convert them to vectors.

Reply:
I followed the previous implementation, But I removed the vector conversion and it works fine.
Thanks!